workflows/dags/author/author_create/author_create_init.py

```diff
@@ -17,17 +17,22 @@
 logger = logging.getLogger(__name__)
 
 
+def task_failure_alert(context):
+    print(f"Task has failed, task_instance_key_str: {context['task_instance_key_str']}")
+
+
 @dag(
     params={
         "workflow_id": Param(type="string", default=""),
         "data": Param(type="object", default={}),
+        "collection": Param(type="string", default=AUTHORS),
```

Comment:
because there's no generic `/api/workflows` api anymore, we have to pass this to know which collection we want to change the workflow of.

Should we create this generic endpoint for this holistic  workflow actions such as setting its status to `ERROR` ?

Reply:
Let's create a task for it

Reply:
https://github.com/cern-sis/issues-inspire/issues/645 . So for the time being I undo this change?

Reply:
No we need it to be able to test it, we can refactor it with the other task